test/cluster/blue-green-deployment/deploy.td

```diff
@@ -9,8 +9,10 @@
 
 > CREATE SCHEMA prod_deploy
 > CREATE MATERIALIZED VIEW prod_deploy.counter_mv IN CLUSTER prod_deploy AS SELECT count(*), 'some new value' FROM counter
+> CREATE MATERIALIZED VIEW prod_deploy.counter_mv2 IN CLUSTER prod_deploy WITH (REFRESH EVERY '60 seconds') AS SELECT count(*), 'some new value' FROM counter
 > CREATE DEFAULT INDEX IN CLUSTER prod_deploy ON counter
 > CREATE DEFAULT INDEX IN CLUSTER prod_deploy ON prod_deploy.counter_mv
+> CREATE DEFAULT INDEX IN CLUSTER prod_deploy ON prod_deploy.counter_mv2
```

Comment:
Does this test involve restarts? If yes, then please don't create indexes on REFRESH MVs for now, due to the problem explained in the "Temporary Limitations" section of the original PR https://github.com/MaterializeInc/materialize/pull/23819

Reply:
It doesn't, but removed anyway.

Reply:
Thx!